test/unit/io_clock.js

```diff
@@ -0,0 +1,34 @@
+var test = require('tap').test;
+var Clock = require('../../src/io/clock');
+var Runtime = require('../../src/engine/runtime');
+
+test('spec', function (t) {
+    var rt = new Runtime();
+    var c = new Clock(rt);
+
+    t.type(Clock, 'function');
+    t.type(c, 'object');
+    t.type(c.projectTimer, 'function');
+    t.type(c.pause, 'function');
+    t.type(c.resume, 'function');
+    t.type(c.resetProjectTimer, 'function');
+    t.end();
+});
+
+test('cycle', function (t) {
+    var rt = new Runtime();
+    var c = new Clock(rt);
+
+    t.ok(c.projectTimer() <= 0.001);
```

Comment:
It could which is why this is not `0`. These worst case scenario I was able to get was `0.001` but it may make more sense to give it a margin of error closer to 100ms (`0.1`).